<launch>
  <node pkg="final"
        type="obstacle_avoidance_node.py"
        name="obstacle_avoidance_node"
        output="screen">
  </node>
</launch>
